plugins/webp-uploads/hooks.php

```diff
@@ -99,12 +103,16 @@ function webp_uploads_create_sources_property( array $metadata, int $attachment_
 	// Create the sources for the full sized image.
 	foreach ( $valid_mime_transforms[ $mime_type ] as $targeted_mime ) {
 		// If this property exists no need to create the image again.
-		if ( ! empty( $metadata['sources'][ $targeted_mime ] ) ) {
+		if (
+			isset( $metadata['sources'][ $targeted_mime ] ) &&
+			is_array( $metadata['sources'][ $targeted_mime ] ) &&
+			0 !== count( $metadata['sources'][ $targeted_mime ] )
+		) {
```

Comment:
Same as above applies here:
```suggestion
		if (
			isset( $metadata['sources'][ $targeted_mime ] ) &&
			is_array( $metadata['sources'][ $targeted_mime ] )
		) {
```